subsys/net/lib/nrf_cloud/src/nrf_cloud_log.c

```diff
@@ -319,7 +319,7 @@ void nrf_cloud_log_enable(bool enable)
 		logs_backend_enable(enable);
 #endif
 		enabled = enable;
-		LOG_DBG("enabled = %d", enabled);
+		LOG_INF("Changing cloud logging enabled to:%d", enabled);
```

Comment:
```suggestion
		LOG_DBG("Changing cloud logging enabled to:%d", enabled);
```

Reply:
This is only logged when the user changes the level compared to what the firmware was built for. This will rarely be logged and is not an issue to be INF. It will improve the user experience so they can see that a change they requested actually happened, especially for our samples. Same for alerts.